### PR TITLE
Persist network graph

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -426,13 +426,27 @@ async fn start_ldk() {
...
 		None::<Arc<dyn chain::Access + Send + Sync>>,
 		logger.clone(),
+		network_graph,
 	));
+	let router_persist = Arc::clone(&router);
+	tokio::spawn(async move {
+		let mut interval = tokio::time::interval(Duration::from_secs(60));
+		loop {
+			interval.tick().await;
+			// We ignore persistence errors here. They are non-fatal as we can just fetch the
```
Comment (valentinewallace) [inline src/main.rs:442]: log warning maybe? could affect UX but be non-obvious

**Review by valentinewallace — APPROVED** [review]
Tested, seems to work!

File: src/main.rs (hunk 1 above)
Comment (TheBlueMatt) [reply src/main.rs]: Done!

**Review by valentinewallace — APPROVED** [review]

**Comment by TheBlueMatt** [comment]
Squashed without diff:
```
$ git diff-tree -U1 873f9db 6200c11
$
```

**Pull request merged**